app/controllers/rentals_controller.rb

```diff
@@ -71,6 +71,7 @@ def rental_schedule
 
   # POST /rentals
   def create
```

Comment:
I know you stepped into a section of code that is untested, but if you could address that, we would very much appreciate it!